packages/versioning/src/validate.ts

```diff
@@ -532,24 +532,6 @@ function validateOperationParameter(
       return;
     }
   }
-
-  if (paramAvailability !== undefined) {
-    validateAvailabilityForContains(
-      program,
-      operationAvailability,
-      paramAvailability,
-      operation,
-      parameter
-    );
-  } else if (paramTypeAvailability !== undefined) {
-    validateAvailabilityForRef(
-      program,
-      operationAvailability,
-      paramTypeAvailability,
-      operation,
-      parameter.type
-    );
-  }
```

Comment:
For this to work, I would actually need to build the availability map based on ALL references in the spec, which would be considerably more design and implementation effort. The new test showcases the problem. Thus, I've just removed this check.